lib/matplotlib/backends/web_backend/mpl.js

```diff
@@ -394,9 +413,10 @@ mpl.findpos = function(e) {
 mpl.figure.prototype.mouse_event = function(event, name) {
     var canvas_pos = mpl.findpos(event)
 
-    if (this.focus_on_mouseover && name === 'motion_notify')
+    if (name === 'button_press')
```

Comment:
My take: we switched to a more Qt-like focus policy, in that the figure takes focus immediately and upon clicking.  *When* the figure has focus, the status bar receives updates.